frontend/public/components/nav/_nav-header.scss

```diff
@@ -2,11 +2,6 @@
   border-bottom: 1px solid $color-pf-black;
   padding: var(--pf-global--spacer--md) var(--pf-global--spacer--sm);
 
-  @media screen and (min-width: $grid-float-breakpoint) {
```

Comment:
We want to keep this so the header vertical gutters maintain the same proportions as the nav and page contents.

Reply:
OK, updated. Thanks!

Reply:
OK, updated. Thanks.